MyFiles/Class.cs

```diff
@@ -0,0 +1,63 @@
+﻿//Class.cs
+
+using System.ComponentModel.Design;
+
+namespace VendingMachineManager
+{
+    public class VendingMachine
+    {
+        private string _product;
+        private string[] Menu = new string[5];
+
+        public void UpdateMenu() //method to update the menu.
```

Comment:
As a recommendation, use triple slash comments. So instead of:
```csharp
public void UpdateMenu() //method to update the menu.
{
...
}
```
Do this instead:
```csharp
/// <summary>
/// Method to update the menu.
/// </summary>
public void UpdateMenu()
{
...
}
```
After doing this, notice what happens when you hover your cursor over `UpdateMenu`. You should see the comment in the code hint. Any place that you invoke `new VendingMachine().UpdateMenu()` (called a "call site") will retain that code hint.

This is really helpful as your application grows. It helps you and other devs have the code hint to tell you some of the implementation details of the code in the method.

https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/recommended-tags

Reply:
FYI, you can add triple slash (XML comments) to any class members, including attributes like `_product` and `Menu` above.